suites/reef/cephadm/tier2-osd-scenarios.yaml

```diff
@@ -152,3 +152,10 @@ tests:
       module: test_verify_incorrect_mem_usage.py
       polarion-id: CEPH-83584007
       abort-on-fail: true
+
+  - test:
```

Comment:
Hi @sayaleeraut 

Can we add same test from squid as well.

Rest look good

Thanks,
Mohit

Reply:
Done @MohitBis , I have added the test in the Quincy and Squid suite as discussed.